framewirc/messages.py

```diff
@@ -124,13 +124,35 @@ def chunk_message(message, max_length):
     return list(_chunk_message(message, max_length))
 
 
-def make_privmsgs(target, message, third_person=False):
-    """Turn a string into a number of PRIVMSG commands."""
-    overhead = 5  # Two spaces, a colon, and the \r\n line ending.
+def make_privmsgs(target, message, third_person=False, mask_length=None):
+    """
+    Turns `message` into a list of `PRIVMSG` commands (to `target`).
+
+    The `third_person` flag can be used to send `/me` commands.
+
+    If the message is too long, we span it across multiple commands.
+
+    We don't send a mask prefix, but the network will add it. This lets clients
+    know who the sender of the message is, and impacts the maximum length of
+    the transmitted command.
+
+    When the `mask_length` is `None`, we allow a default of 100 chars.
+    """
+    # If we don't know exactly how long the mask will be, make a guess.
+    # I can't find a maximum length in the spec; 100 chars seems safe.
+    if mask_length is None:
+        mask_length = 100
```

Comment:
Deserves a better comment... The maximum lengths of _some_ of the values is known, at least. Also, I suspect the `ident` might have a max value of `512`... but that can't be right... right?

Reply:
I wish I could remember where I got any information about this. I guess this comment will have to do for the moment.